CHANGELOG.md

```diff
@@ -1,11 +1,61 @@
+# 4.0.0 (2020-05-29)
+
+### New features
+
+- Added `SkyDocsHeadingAnchorModule`, `SkyDocsMarkdownModule`, `SkyDocsModuleInfoModule`, and `SkyDocsSafeHtmlModule` to the exports API. [#57](https://github.com/blackbaud/skyux-docs-tools/pull/57)
```

Comment:
I don't see any of these items in the code changes in that PR. Is this the correct PR?